### DISCUSSION TITLE
Self-hosting and Day-passes

### DISCUSSION THREAD

**Discussion by jmlaurenti** [Budibase/budibase#8617] [General]
Hi,
kudos to the team for the quality of your work ! :-)

I'm currently self-hosting Budibase, and while I fully understand subscribing to pricing-plans to unlock additional features (like backuping apps), I don't really understand day-passes concept (again, in self-hosting context).

I'm interested in Pro plan to unlock backup feature, but do you confirm that, when using my own instance, I have to take the Team subscription as soon as I have more than 4 regular users (ie more than 75days of usage per month) ? 

Thanks for the clarification.

Best regards,
JML

**Comment by mjashanks** [comment]
Hey JML!

Happy to have a chat about pricing - could we do a [quick call](https://meetings-eu1.hubspot.com/joe-moss/call-with-joe-moss-and-mike)?

**Comment by drearystate** [comment]
I am curious about this as well. I just started with Budibase and I host my own with the free unlimited users. I would like to pay for the additional features but not at the cost to limiting my user count.
